tests/workspace_test.go

```diff
@@ -85,3 +85,17 @@ func TestGetWorkspaceRepository(t *testing.T) {
 		t.Error("Cannot catch repos full name.")
 	}
 }
+
+func TestGetWorkspacePermissionForUser(t *testing.T) {
+	c := getBitbucketClient(t)
+	workspaceName := getWorkspace(t)
+
+	res, err := c.Workspaces.Permissions.GetUserPermissions(workspaceName, "josemiguelmelo")
```

Comment:
Did you mean to leave your nickname hard-coded in this test?

Reply:
Oh sorry I overlooked. @sl4mmy